HealthcareBase/Service/HospitalResourcesService/EquipmentService/EquipmentService.cs

```diff
@@ -57,7 +58,7 @@ public void Delete(EquipmentUnit equipmentUnit)
             equipmentUnitRepository.Repository.Delete(equipmentUnit);
         }
 
-        private void DeleteFromHospitalizations(EquipmentUnit equipmentUnit)
+        private void DeleteFromHospitalizations(EquipmentUnit equimpentUnit)
```

Comment:
Rename parameter again :)

Reply:
This method was there before me, so I didn't want to touch it, but I am pretty sure that no one needs it anymore, so I will delete it.